miner/Miner.hs

```diff
@@ -125,21 +138,25 @@ data Env = Env
     { envGen :: !MWC.GenIO
     , envMgr :: !Manager
     , envLog :: !LogFunc
+    , envCmd :: !Command
     , envArgs :: !ClientArgs
     , envHashes :: IORef Word64
-    , envSecs :: IORef Word64 }
+    , envSecs :: IORef Word64
+    , envSuccessInitTime :: IORef Int
+    , envUrls :: MVar (NESeq (T2 BaseUrl ChainwebVersion)) }
```

Comment:
Make this an `IORef` to unify it with the other fields and reduce the number of imports necessary.